packages/hint/src/lib/config.ts

```diff
@@ -253,17 +252,55 @@ export class Configuration {
         return browserslist(config.browserslist);
     }
 
+    private static groupHintConfigs(configs: ConfigFiles, file: string): ConfigFiles {
+        const basename = path.basename(file) as ConfigFile;
+
+        if (!basename) {
+            throw new Error(`unrecognised config file '${basename}'`);
+        }
+
+        const hintConfig = Configuration.loadConfigFile(file);
+
+        if (basename === ConfigFile.PackageJson) {
+
+            const packageJson = loadJSONFile(file);
```

Comment:
Since `Configuration.loadConfigFile` already handles loading `package.json` I'm confused as to why this separate load is needed. Is there a particular reason this needs to be special-cased that I'm missing?

Reply:
The way I understand it, it's possible to declare browserslist in both the root of the package.json and in the hintConfig prop in the package.json. So I'm trying to catch the case if browserslist has been declared twice in package.json, once in the root and once in the hintConfig prop.